packages/target-tauri/src-tauri/src/html_window/email_scheme.rs

```diff
@@ -34,6 +34,15 @@ pub(crate) fn email_protocol<R: tauri::Runtime>(
     // URI format is email://<anything>
     // (file path doesn't matter, because the html content is always returned)
 
+    let webview_label = ctx.webview_label();
+    if !webview_label.starts_with("html-window:") || !webview_label.ends_with("-mail") {
+        error!(
+            "prevented other window from accessing email:// scheme (webview label: {})",
+            webview_label
+        );
+        return;
+    }
+
```

Comment:
technically this unnecessary, because this scheme loads the email based on the window/webview it was accessed by, so in the case of another window accessing it it will return an error already.

Reply:
This is the case right now, but hypothetically the behavior could change.
So I think it's a safe bet to just have a simple guard at the very start of the function.
This also helps with readability. It's media to clear that this protocol is only supposed to be used by windows with such label.